Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -213,7 +213,6 @@ public struct TuistCommand: AsyncParsableCommand {
             }
             ServiceContext.current?.ui?.error(.alert(errorAlert.message, nextSteps: errorAlertNextSteps))
         } else if let successAlert = successAlerts.last {
-            print("\n")
```

Comment:
I personally like the success message better without the extra newline cc @pepicrft 
<img width="1128" alt="image" src="https://github.com/user-attachments/assets/6d70903b-8e42-4c77-8728-0e6fca13b97d" />